hub/core/version_control/commit_diff.py

```diff
@@ -1,32 +1,34 @@
-from typing import Set
+from typing import Set, List
 from hub.core.storage.cachable import Cachable
 
 
 class CommitDiff(Cachable):
     """Stores set of diffs stored for a particular tensor in a commit."""
 
-    def __init__(self, created=False) -> None:
+    def __init__(self, first_index=0, created=False) -> None:
         self.created = created
-        self.data_added: Set[int] = set()
+        self.data_added: List[int] = [first_index, first_index]
         self.data_updated: Set[int] = set()
+        self.info_updated = False
```

Comment:
for now this will always be False. I've added this to the format so we don't break backwards compatibility in the future.